from robo_sdk.robo_ai.exception.api_error import ApiError


class InvalidCredentialsError(ApiError):
    pass
